specs/Spec.Blake2.fst

```diff
@@ -491,6 +491,17 @@ let blake2_update a kk k d s =
      else blake2_update_blocks a (size_block a) d s
   else blake2_update_blocks a 0 d s
 
+val blake2_key_block:
```

Comment:
can this definition be reused in blake2_update_key above?

Reply:
Seems like it. Should I make the change in this PR?

Reply:
if you don't mind, that's slightly cleaner, and justifies why this function is in the main spec file (right now it's defined but unused in the main spec)